### PR TITLE
Update layout

### PR DESCRIPTION
**Pull request by ShellyXueHan**
1. the hint for restarting registration process in registration page
2. the button to invite new user in registered page

/bot-ignore-length

### CODE REVIEW

File: src/components/UI/SideMessages.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
+.message-group {
+  display: flex;
+  flex-flow: row wrap;
+}
+
+.center-message {
+  flex-grow: 4;
```
Comment (patricksimonian) [inline src/components/UI/SideMessages.css]: question on these messages, Are they meant to grow and shrink? Your left and right messages appear to be fixed in width. Did you want them to have an initial width (kind of like a min-width) and grow from there if there is extra space?

File: src/components/UI/SideMessages.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+  .right-message {
+    width: 150px;
+    flex-grow: 1;
+    border-left: none;
+    border-top: 1px solid #e5e5e5;
+    margin-top: 50px;
+    padding-top: 20px;
+  }
+}
```
Comment (patricksimonian) [inline src/components/UI/SideMessages.css]: new line

File: src/containers/RocketChat.css [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+}
+
+.invite-message p {
+  margin: 10px 0 !important;
+}
+
+.invite-button {
+  margin: auto 5px 0 5px;
+  width: 150px;
+}
```
Comment (patricksimonian) [inline src/containers/RocketChat.css]: new line

**Review by patricksimonian — APPROVED** [review]

**Comment by ShellyXueHan** [comment]
@patricksimonian issue with #34 is fixed in this pr.

**Pull request closed without merging**